bindings/python/examples/first_frame_network/readme.md

```diff
@@ -0,0 +1,65 @@
+# First frame Example
```

Comment:
This readme talks about too much code which is already done in the first_frampe.py example. It would be better to talk only about getCameraAtIp(). Also it would be nice to give an example of how it can be used (use a real ip).